ee/api/test/test_capture.py

```diff
@@ -0,0 +1,145 @@
+import json
+from typing import Any
+from unittest.mock import patch
+
+from django.test.client import Client
+from rest_framework import status
+
+from ee.kafka_client.topics import KAFKA_EVENTS_PLUGIN_INGESTION
+from posthog.test.base import APIBaseTest
+
+
+def mocked_get_team_from_token(_: Any) -> None:
+    raise Exception("test exception")
+
+
+class TestCaptureAPI(APIBaseTest):
+    def setUp(self):
+        super().setUp()
+        self.client = Client()
+
+    @patch("ee.kafka_client.client._KafkaProducer.produce")
+    def test_produce_to_kafka(self, kafka_produce):
```

Comment:
this is just a basic capture endpoint test for EE/CH deployments. We didn't really have tests for this, only Postgres